client/src/components/App.jsx

```diff
@@ -1,24 +1,50 @@
+/* eslint-disable import/extensions */
 import React from 'react';
-// eslint-disable-next-line import/extensions
 import ReviewTile from './ReviewTile/ReviewTile.jsx';
 import Overview from './Overview/Overview.jsx';
 import RelatedItems from './RelatedItems/RelatedItems.jsx';
 import QandA from './QandA/QandA.jsx';
 // eslint-disable-next-line import/extensions
 import products from '../../dummydata.js';
+import queries from './queries.js';
 
 class App extends React.Component {
   constructor() {
     super();
-    this.state = { items: products };
+    this.state = {
+      items: products,
+      productList: [],
+      productInfo: {},
+      productStyles: {},
+      relatedProducts: [],
+      currentProduct: 17762,
```

Comment:
nice job starting with the first product since there is no page to navigate from :)